src/stringnumber.js

```diff
@@ -27,7 +27,11 @@ export class StringNumber extends Modifier {
     let shift_left = 0;
     let shift_right = 0;
 
-    let i, num, note, pos, props_tmp;
+    let i;
+    let num;
+    let note;
+    let pos;
+    let props_tmp;

```

Comment:
I'd say this block is in need of a little refactor anyway.  Whether these are one line or multiple lines, perhaps they shouldn't be there at all, and rather declared when needed (that's what `let` is for!)